etl/__init__.py

```diff
@@ -7,6 +8,5 @@
 else:  # linux or other
     p_drive = pathlib.Path("/p/")
 
-# write test for proj==""
-abs_proj_path = pathlib.Path(__file__).parent.parent
-proj = pathlib.Path(__file__).parent.relative_to(abs_proj_path)
+rel_root = pathlib.Path(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Sure, if Zarr only raises OSErrors. If they would raise zarrspecificerror you could only catch that.

Reply:
I'm going to merge this branch into the main, to avoid that the main becomes too outdated, and to avoid consistency issues with the front-end. Based on the reply by Floris on this discussion, I have the impression that this is resolved. If not, please follow-up later.